app/routes/api/github/get-proyectActivity.tsx

```diff
@@ -38,7 +39,7 @@ export const getActivity = async (repo: string, projectId: string) => {
   
     
     }catch(e){
-      console.error(e);
+      console.error('no existe la pagina', repoUrlClean);
```

Comment:
LGTM, but maybe you could put the error message in English to keep the whole project in one language. But I don't know if it's necessary because it's an error message jajaja.